Core/src/main/java/com/plotsquared/core/plot/flag/GlobalFlagContainer.java

```diff
@@ -153,6 +154,7 @@ private GlobalFlagContainer() {
         this.addFlag(DeviceInteractFlag.DEVICE_INTERACT_FALSE);
         this.addFlag(DisablePhysicsFlag.DISABLE_PHYSICS_FALSE);
         this.addFlag(DropProtectionFlag.DROP_PROTECTION_FALSE);
+        this.addFlag(EditSignFlag.EDIT_SIGN_FALSE);
```

Comment:
Is it worth only applying the flag for > 1.20.1?

Reply:
I think this might cause issues when trying to access the flag value on other versions, which normally shouldn't happen, but I don't think it's worth it (and we also didn't do that for other flags previously).